src/TodoList.js

```diff
@@ -17,12 +17,13 @@ const todoList = [
     }
   ];
   
-const TodoList = () => {
+const TodoList = ({todoList}) => {
+  const {id, title} = todoList;
```

Comment:
nice job using a destructuring assignment for these properties, but they're happening in slightly the wrong place. The `todoList` is an array, and doesn't contain an `id` or `title` key that you can destructure, so both of these are being set to `undefined`

This leads to a console error complaining that all the `TodoListItems` are being assigned the same `key`, since they all have the same `id`

Reply:
```suggestion
```